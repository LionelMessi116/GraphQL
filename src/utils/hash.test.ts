import { hashString } from './hash';

describe('hash', () => {
  it('should returned a murmur hashed string from a query string', () => {
    expect(hashString(`{ todos { id } }`)).toMatchSnapshot();
  });
});
