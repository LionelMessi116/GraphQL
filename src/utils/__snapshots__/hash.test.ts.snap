// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hash should returned a murmur hashed string from a query string 1`] = `"1rvkz44"`;
